src/components/widgets/SelectWidget.js

```diff
@@ -9,21 +9,19 @@ import { asNumber } from "../../utils";
  */
 function processValue(schema, value) {
   if (schema.type === "array") {
-    let valueType = schema.items.type;
+    const valueType = schema.items.type;
     if (valueType === "number" || valueType === "integer") {
       return value.map(v => asNumber(v));
     }
     return value;
   }
-  else {
-    let valueType = schema.type;
-    if (valueType === "boolean") {
-      return value === "true";
-    } else if (valueType === "number") {
-      return asNumber(value);
-    }
-    return value;
+  const valueType = schema.type;
+  if (valueType === "boolean") {
+    return value === "true";
+  } else if (valueType === "number") {
+    return asNumber(value);
   }
+  return value;
```

Comment:
Actually I suspect we could rewrite the whole function in a more compact fashion, like:

```js
function processValue({type, items}, value) {
  if (type === "array" && items && ["number", "integer"].includes(items.type)) {
    return value.map(asNumber);
  } else if (type === "boolean") {
    return value === "true";
  } else if (type === "number") {
    return asNumber(value);
  }
  return value;
}
```